crypto/crypto.go

```diff
@@ -1,14 +1,18 @@
 package crypto
 
 import (
-	"github.com/tendermint/tendermint/crypto/tmhash"
+	"crypto/sha256"
+
 	"github.com/tendermint/tendermint/internal/jsontypes"
 	"github.com/tendermint/tendermint/libs/bytes"
 )
 
 const (
+	HashSize      = sha256.Size
+	HashBlockSize = sha256.BlockSize
```

Comment:
I don't think we need this, do we?

Reply:
fair enough. 

a lot of this gets used in the SDK